OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseTabsDashboardFragment.java

```diff
@@ -129,6 +137,18 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
         }
     }
 
+    private View.OnClickListener onCloseClick() {
+        return v -> requireActivity().finish();
+    }
+
+    private View.OnClickListener onFindCourseClick() {
+        return v -> {
+            environment.getAnalyticsRegistry().trackUserFindsCourses();
+            EventBus.getDefault().post(new MoveToDiscoveryTabEvent(Screen.DISCOVERY));
```

Comment:
The user not redirecting to the discovery page in response to a button click.

Reply:
working fine on my side